drivers/SmartThings/zigbee-smoke-detector/capabilities/selfCheck.yaml

```diff
@@ -0,0 +1,45 @@
+id: stse.selfCheck
+version: 1
+status: proposed
+name: Self Check
+ephemeral: false
+attributes:
+  selfCheckState:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - idle
+            - selfChecking
+            - selfCheckCompleted
+      additionalProperties: false
+      required:
+        - value
+    setter: setSelfCheckState
+  supportedSelfCheckState:
+    schema:
+      type: object
+      additionalProperties: false
+      properties:
+        value:
+          type: array
+          items:
+            type: string
+            enum:
+              - idle
+              - selfChecking
+              - selfCheckCompleted
+commands:
+  setSelfCheckState:
+    name: setSelfCheckState
+    arguments:
+      - name: state
+        optional: false
+        schema:
+          type: string
+          enum:
+            - idle
+            - selfChecking
+            - selfCheckCompleted
```

Comment:
The way you've used this it seems like it should just be a single command to `startSelfCheck`.

Reply:
@Hongming6

Reply:
> The way you've used this it seems like it should just be a single command to `startSelfCheck`.

Hi @Hongming6

Per our discussion. Aqara hopes DeviceCard could display these 3 difference selfCheck status, am I right ?

Reply:
yes! The user needs to know the current device self-test progress from the device card.

Reply:
The state should be reported by the attribute. This is the command. 

The current driver doesn't even support the ability to set the state to "idle" or "selfCheckCompleted", which is why I think this command accepting these values is not necessary.

Reply:
The current driver  support the ability to set the state to "idle" or "selfCheckCompleted".Just don't support selfChecking.
We added selfChecking to show that the device is performing an action within the APP.